src/Commands/GenerateDocumentation.php

```diff
@@ -79,7 +79,7 @@ private function writeMarkdown($parsedRoutes)
 
         $infoText = view('apidoc::partials.info')
             ->with('outputPath', ltrim($outputPath, 'public/'))
-            ->with('showPostmanCollectionButton', config('apidoc.postman'));
+            ->with('showPostmanCollectionButton', config('apidoc.postman.enabled') ?? config('apidoc.postman'));
```

Comment:
Okay, I realize you're doing this to maintain BC. But we need to note a few things:
- `config()` will always return a value or null, so the other end of the `??` will never be called.
- since [`config` accepts a default](https://laravel.com/docs/5.7/helpers#method-config), I suggest doing this instead:

```php
config('apidoc.postman.enabled', is_bool(config('apidoc.postman')) ? config('apidoc.postman') : false);
```
And encapsulate that in a function (probably `shouldGeneratePostmanCollection()`) so you don't have to repeat it everywhere.

Reply:
Okay I fixed that now. Do you also agree with my update to force the URL in the postman collection? This has to do with the fact that [the URL generator is not updated](https://github.com/laravel/framework/issues/18613).

Reply:
@shalvah The other end of `??` will be called if the first end is null. `??` is a null coalescing operator. So, it should be fine just the way it is.

```suggestion
            ->with('showPostmanCollectionButton', config('apidoc.postman.enabled') ?? config('apidoc.postman'));
```

Reply:
Alright @ezra-obiwale , but extracting into a new method and using `config()`'s functionality is better is better.
@ewerkema I think that's okay.